apps/stack/template.yaml

```diff
@@ -310,7 +346,15 @@ Resources:
         - !Ref Topic
       Parameters:
         BucketArn: !GetAtt Bucket.Arn
-        FilterUri: !Ref MetricStreamFilterUri
+        FilterUri: !If
+          - DeployLambda
+          - 's3://observeinc/cloudwatchmetrics/filters/default.yaml'
```

Comment:
does the lambda ever use this file? if not does this even need to be a real s3 file?

Reply:
also we should write some comment here to explain why this is necessary